libs/ml/include/ml/optimisation/adam_optimiser.hpp

```diff
@@ -17,16 +17,9 @@
 //
 //------------------------------------------------------------------------------
 
-#include "math/base_types.hpp"
-#include "math/standard_functions/pow.hpp"
-#include "math/standard_functions/sqrt.hpp"
 #include "ml/core/graph.hpp"
```

Comment:
I think you can move this include to the source code and just forward declare graph, right?

Reply:
i think this is true for all of these optimisers too right?

Reply:
but I use Graph here. If I remove it from adam_optimiser.hpp  I will have:  error: use of undeclared identifier 'Graph'